File: Proiectul_meu/Models/Sosete.cs
﻿using Proiectul_meu.Models.Base;
using System.ComponentModel.DataAnnotations.Schema;

namespace Proiectul_meu.Models
{

    public class Sosete : Baza
    {
        public string Descriere { get; set; }

        public string Culoare { get; set; }

        public string Marime { get; set; }

        public string Material { get; set; }

        public double Pret { get; set; }

        public bool Scurte { get; set; }

        public bool Diferite { get; set; }

        public Guid TreningId { get; set; }

        [ForeignKey("TreningId")]
        public Trening Trening { get; set; }
    }
}

File: Proiectul_meu/ClientApp/src/app/models/Sosete.ts
import { Trening } from "./Trening";

export interface Sosete {
  id?: string;
  descriere: string;
  culoare: string;
  marime: string;
  material: string;
  pret: number;
  scurte: boolean;
  diferite: boolean;
  treningId?: string;
  trening: Trening;
}
